src/components/app/WindowTitle.js

```diff
@@ -72,6 +72,9 @@ class WindowTitleImpl extends PureComponent<Props> {
             title += formattedMetaInfoString + SEPARATOR;
           }
           title += _formatDateTime(meta.startTime);
+          if (meta.endTime) {
+            title += SEPARATOR + _formatDateTime(meta.endTime);
+          }
```

Comment:
Is the end time so useful that it should go in the title? The titles of your profiles are already really long, this makes them even longer. The long titles compete for space with the other buttons in that row. For example, once you zoom into time range selections, more buttons get stacked into the same row and they end up in an odd place when the title is so long.

Reply:
You're right, I remove it